src/style.css

```diff
@@ -0,0 +1,130 @@
+body,
+p {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+}
```

Comment:
move font-family to html, box-sizing to *
and all this styles to reset.css file